src/main/resources/liquibase.xml

```diff
@@ -555,5 +555,29 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet id="202007060140" author="Neeraj" context="POC">
+        <addColumn tableName="patient">
+            <column name="date_of_birth" type="BIGINT">
```

Comment:
Why are we storing DOB as a BIGINT not as a DateTime?

Reply:
it just contains date, 
 not month and year